### ISSUE TITLE
'Translation of' dropdown field in segment emails is incorrectly showing template emails

### ISSUE THREAD

**Issue by Rixita** [mautic/mautic#5135]
What type of report is this:

| Q  | A
| ---| ---
| Bug report? | Yes
| Feature request? | 
| Enhancement? | 

## Description:
When a segment email is set to be "a translation of", you save it and go back to edit that email. The setting you saved before -> Is translation of" the parent email -> is not present anymore.
Now if you would change something else and save the email. It forgot that it was a translation of email and you have to set it again. 

For the "Template Emails" this seems to work well.

## If a bug:

| Q   | A
| --- | ---
| Mautic version | 
| PHP version | 

### Steps to reproduce:
1. Create a parent segment email for ex. "Email Welcome"
2. Set language to english
3. Create a new segment email for ex. "Email Willkommen"
4. Choose a language - for ex. German
5. Set "is translation of" to the previous created segment email "Email Welcome"
6. Save & Close 
7. Open / edit the child email "Email Willkommen"
8. You will notice that the setting "is translation of" is empty.
9 If you now Save & Close, you will see the Child email is no longer set as a child email / translation of the parent email  "Email Welcome"
 
### Log errors: 

_Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable._

**Comment by wucherpfennig** [comment]
Same here. Latest version

**Comment by wucherpfennig** [comment]
This can be reproduced in 2.14.1. It seems to happen if you clone the parent and assign the cloned version later on to the correct parent.

WORKAROUND: after editing the translation set the correct translation parent and save it as usual. The link will be create although not displayed in the front end (check the parents translation tab). If you need to change the child, set the translation parent **AGAIN**. double + tripple check after saving helps

**Comment by wucherpfennig** [comment]
still an issue. latest version

**Comment by wucherpfennig** [comment]
still an issue. latest version (2.15.1)

**Comment by RCheesley** [comment]
Hi there, as we are only providing fixes for Mautic 3.x going forward, can you confirm if this continues in Mautic 3.0.1?
Thanks!

**Comment by wucherpfennig** [comment]
Well if there were no changes in v3 (where?) This issue persists since almost 3 years.

**Comment by RCheesley** [comment]
Hi there, I tested this in a Mautic 3.0.1 instance and I don't think that it is related to the translation per say, but the fact that the dropdown seems to only be (incorrectly) pulling in campaign emails.

I was able to successfully set the translated email but as reported, when you go back and edit it, you do not see the translated email in the dropdown despite it being correctly associated as a translation.

Here's a short screencast to help with reproducing and hopefully fixing the bug https://youtu.be/p_VCTRfKioI.

Thanks for your help and for the workaround @wucherpfennig, here's hoping that we can get this fixed asap!
